client/webserver/site/src/js/markets.js

```diff
@@ -860,6 +887,7 @@ export default class MarketsPage extends BasePage {
     const lots = Math.floor(order.qty / lotSize)
     const adjusted = lots * lotSize
     page.lotField.value = lots
+    this.previewOrder()
     if (!order.isLimit && !order.sell) return
     if (finalize) page.qtyField.value = (adjusted / 1e8)
```

Comment:
`previewOrder` reads the `page.qtyField`, which could be adjusted two lines down. `previewOrder` must go after that. 

`previewOrder` also needs to be called at the end of `lotChanged`.